README.md

```diff
@@ -1,6 +1,6 @@
 # node-oracledb version 1.4
 
-## <a name="about"></a> 1. About node-oracledb
+## <a name="about"></a> About node-oracledb
```

Comment:
Another point in your favor is that this is only a brief readme, and not so subject to the 'maze of twisty little passage' problem that can occur when trying to point people to sections of longer doc.